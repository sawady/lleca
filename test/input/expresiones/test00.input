(1 * 22 + 333 * 4444) div (1 * (22 + 333))
