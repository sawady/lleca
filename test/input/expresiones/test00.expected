div(
  add(
    mul(
      1,
      22
    ),
    mul(
      333,
      4444
    )
  ),
  mul(
    1,
    add(
      22,
      333
    )
  )
)
